webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -27,5 +27,9 @@ export const get = async (
       payload: modifiedPayload,
       response: error,
     });
+
+    if (error.response.status === 401) {
```

Comment:
Foreman may return with unauthorized status in some cases (afaik,  when admin permissions required) so these cases should be filtered out, or this error handling should be part of a specific action